### ISSUE TITLE
onCanvasClick not listening

### ISSUE THREAD

**Issue by RichardWeug** [namespace-ee/react-calendar-timeline#383]
**Describe the bug**
When I click on empty canvas, "onCanvasClick" doesn't get called.

**To Reproduce**

```
<Timeline
  onCanvasClick={() => console.log("test")}
  groups={employees}
  items={items}
  defaultTimeStart={moment().add(-12, 'hour')}
  defaultTimeEnd={moment().add(12, 'hour')}
/>
```

**Expected behavior**
The console logs "test".

**Library Version**
0.18.0

**Desktop (please complete the following information):**

* OS: Chrome OS
* Browser: Chrome
* Version: 69.0.3494.0 (Official Build) dev (64-bit)

**Additional context**
I've taken a look at issue #374 beforehand. Even the test code that @mcMickJuice posted doesn't console log anything for me.

**Comment by RichardWeug** [comment]
This issue only occurs on version 0.18.0! I rolled back to 0.17.3 and now it works perfectly.

**Comment by Ilaiwi** [comment]
this is related to this #378 
The `MarkerCanvas` is blocking the event trigger

**Comment by mcMickJuice** [comment]
Fixed in 0.18.2
